circleci: failed tests (15817): TestRepair

The following test appears to have failed:

[#15817](https://circleci.com/gh/cockroachdb/cockroach/15817):

```
  trace:     https://127.0.0.1:32888/debug/requests
  logs:      /tmp/circle-artifacts.xDl4Fmw/acceptance/TestRepair/roach2/cockroach.INFO
  pprof:     docker exec -it 2ee2d /bin/bash -c 'go tool pprof /cockroach <(wget --no-check-certificate -qO- https://$(hostname):8080/debug/pprof/heap)'
  cockroach: [start --ca-cert=/certs/ca.crt --cert=/certs/node.crt --key=/certs/node.key --host=roach2 --alsologtostderr=INFO --store=path=/data3.0 --join=roach0:26257 --log-dir=/logs/roach2 --logtostderr=false]
I160404 16:58:19.316934 acceptance/replication_test.go:50  waiting for first range to have 3 replicas
panic: test timed out after 5m0s

goroutine 2387 [running]:
panic(0x117fd00, 0xc8203d0ab0)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:725 +0x14b
created by time.goFunc
	/usr/local/go/src/time/sleep.go:129 +0x3a

goroutine 1 [chan receive]:
testing.RunTests(0x17581b0, 0x1dd3c00, 0x12, 0x12, 0xc82023fd01)
	/usr/local/go/src/testing/testing.go:583 +0x8d2
testing.(*M).Run(0xc8203daf08, 0x17560e0)
	/usr/local/go/src/testing/testing.go:515 +0x81
github.com/cockroachdb/cockroach/acceptance.TestMain(0xc8203daf08)
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:48 +0x3e
main.main()
	github.com/cockroachdb/cockroach/acceptance/_test/_testmain.go:86 +0x114

goroutine 17 [syscall, 4 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1

goroutine 7 [chan receive]:
github.com/cockroachdb/cockroach/util/log.(*loggingT).flushDaemon(0x20aac40)
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:993 +0x64
created by github.com/cockroachdb/cockroach/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/util/log/clog.go:594 +0x8a

goroutine 20 [select, 4 minutes, locked to thread]:
runtime.gopark(0x17589e8, 0xc820028728, 0x14948c0, 0x6, 0x18, 0x2)
	/usr/local/go/src/runtime/proc.go:262 +0x163
runtime.selectgoImpl(0xc820028728, 0x0, 0x18)
	/usr/local/go/src/runtime/select.go:392 +0xa67
runtime.selectgo(0xc820028728)
	/usr/local/go/src/runtime/select.go:215 +0x12
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal1_unix.go:279 +0x358
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1998 +0x1

goroutine 9 [syscall, 4 minutes]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x132
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 10 [chan receive, 4 minutes]:
github.com/cockroachdb/cockroach/acceptance.TestMain.func1()
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:39 +0xd8
created by github.com/cockroachdb/cockroach/acceptance.TestMain
	/go/src/github.com/cockroachdb/cockroach/acceptance/main_test.go:47 +0x30

goroutine 2322 [select]:
github.com/cockroachdb/cockroach/acceptance.checkRangeReplication.func1(0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/acceptance/replication_test.go:53 +0x4b8
github.com/cockroachdb/cockroach/util.RetryForDuration(0x37e11d600, 0xc820393c18, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:125 +0x180
github.com/cockroachdb/cockroach/util.SucceedsSoonDepth(0x1, 0x7f473391fcb8, 0xc82007a090, 0xc820393c18)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:112 +0x3f
github.com/cockroachdb/cockroach/util.SucceedsSoon(0x7f473391fcb8, 0xc82007a090, 0xc820393c18)
	/go/src/github.com/cockroachdb/cockroach/util/testing.go:106 +0x3e
github.com/cockroachdb/cockroach/acceptance.checkRangeReplication(0xc82007a090, 0x7f4733972e00, 0xc8201ce0d0, 0x4a817c800)
	/go/src/github.com/cockroachdb/cockroach/acceptance/replication_test.go:72 +0x226
--
testing.tRunner(0xc82007a090, 0x1dd3d20)
	/usr/local/go/src/testing/testing.go:473 +0x98
created by testing.RunTests
	/usr/local/go/src/testing/testing.go:582 +0x892

goroutine 164 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f4733967720, 0x72, 0xc82040a000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8202ae3e0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8202ae3e0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8202ae380, 0xc82040a000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c270, 0xc82040a000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc82012e820)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 14 [IO wait, 4 minutes]:
net.runtime_pollWait(0x7f47339678a0, 0x72, 0xc820245000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8201a7cd0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8201a7cd0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8201a7c70, 0xc820245000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a360, 0xc820245000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc820246410)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 15 [select, 4 minutes]:
net/http.(*persistConn).writeLoop(0xc820246410)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 565 [select, 2 minutes]:
net/http.(*persistConn).writeLoop(0xc8201ce4e0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 605 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc820324840)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 589 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc8202fee70)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 407 [IO wait, 3 minutes]:
net.runtime_pollWait(0x7f4733967660, 0x72, 0xc820229000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc820314ca0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820314ca0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820314c40, 0xc820229000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c2d8, 0xc820229000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201329c0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 715 [select, 1 minutes]:
net/http.(*persistConn).writeLoop(0xc820246750)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 276 [select, 3 minutes]:
net/http.(*persistConn).writeLoop(0xc8201ce340)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 714 [IO wait, 1 minutes]:
net.runtime_pollWait(0x7f4733922560, 0x72, 0xc8203c7000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203b2370, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203b2370, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203b2310, 0xc8203c7000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a1d8, 0xc8203c7000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc820246750)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 165 [select, 4 minutes]:
net/http.(*persistConn).writeLoop(0xc82012e820)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1219 [select, 1 minutes]:
net/http.(*persistConn).writeLoop(0xc8201ce5b0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 861 [IO wait, 1 minutes]:
net.runtime_pollWait(0x7f47339224a0, 0x72, 0xc8202a5000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203d21b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203d21b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203d2150, 0xc8202a5000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a1f8, 0xc8202a5000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc82012ea90)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 2367 [select]:
github.com/cockroachdb/cockroach/rpc.(*Context).runHeartbeat(0xc8202aed20, 0xc8205363c0, 0xc82046f030, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:207 +0x59b
github.com/cockroachdb/cockroach/rpc.(*Context).GRPCDial.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:156 +0x66
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202aecb0, 0xc820359d40)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62

goroutine 593 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc8202ff080)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 862 [select, 1 minutes]:
net/http.(*persistConn).writeLoop(0xc82012ea90)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 602 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc820011e40)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 564 [IO wait, 2 minutes]:
net.runtime_pollWait(0x7f47339677e0, 0x72, 0xc8204b5000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203156b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203156b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820315650, 0xc8204b5000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a208, 0xc8204b5000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201ce4e0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 275 [IO wait, 3 minutes]:
net.runtime_pollWait(0x7f47339674e0, 0x72, 0xc8201eb000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203141b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203141b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820314150, 0xc8201eb000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a028, 0xc8201eb000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201ce340)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 408 [select, 3 minutes]:
net/http.(*persistConn).writeLoop(0xc8201329c0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 2103 [select]:
net/http.(*persistConn).writeLoop(0xc820246b60)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 628 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc82036e4d0)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 633 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc8203cc0b0)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 646 [chan receive, 3 minutes]:
database/sql.(*DB).connectionOpener(0xc820324370)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 2347 [select]:
net/http.(*persistConn).writeLoop(0xc8202472b0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1218 [IO wait, 1 minutes]:
net.runtime_pollWait(0x7f47339226e0, 0x72, 0xc8204fe000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203b2290, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203b2290, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203b2230, 0xc8204fe000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a210, 0xc8204fe000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201ce5b0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 770 [chan receive, 2 minutes]:
database/sql.(*DB).connectionOpener(0xc8202ff4a0)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 772 [chan receive, 2 minutes]:
database/sql.(*DB).connectionOpener(0xc8203cc160)
	/usr/local/go/src/database/sql/sql.go:727 +0x45
created by database/sql.Open
	/usr/local/go/src/database/sql/sql.go:493 +0x33f

goroutine 1802 [select]:
net/http.(*persistConn).writeLoop(0xc82012f380)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1463 [IO wait]:
net.runtime_pollWait(0x7f47339669a0, 0x72, 0xc8204a3000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203b31e0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203b31e0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203b3180, 0xc8204a3000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c2f8, 0xc8204a3000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201cef70)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1564 [select]:
net/http.(*persistConn).writeLoop(0xc820132340)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1690 [IO wait]:
net.runtime_pollWait(0x7f4733966a60, 0x72, 0xc820298000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8202b01b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8202b01b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8202b0150, 0xc820298000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c178, 0xc820298000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc82012edd0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 2205 [select]:
net/http.(*persistConn).writeLoop(0xc8201ce820)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 2375 [IO wait]:
net.runtime_pollWait(0x7f47339227a0, 0x72, 0xc82011f400)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc820294d10, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc820294d10, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc820294cb0, 0xc82011f400, 0x400, 0x400, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c530, 0xc82011f400, 0x400, 0x400, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
google.golang.org/grpc/transport.(*http2Client).reader(0xc820508780)
	/go/src/google.golang.org/grpc/transport/http2_client.go:756 +0xef
created by google.golang.org/grpc/transport.newHTTP2Client
	/go/src/google.golang.org/grpc/transport/http2_client.go:200 +0x159a

goroutine 1053 [select, 1 minutes]:
net/http.(*persistConn).writeLoop(0xc820246f70)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1052 [IO wait, 1 minutes]:
net.runtime_pollWait(0x7f4733966be0, 0x72, 0xc820398000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203d3330, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203d3330, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203d32d0, 0xc820398000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a2d0, 0xc820398000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc820246f70)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1464 [select]:
net/http.(*persistConn).writeLoop(0xc8201cef70)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 1927 [IO wait]:
net.runtime_pollWait(0x7f4733922ce0, 0x72, 0xc820479000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8202b0920, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8202b0920, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8202b08c0, 0xc820479000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a388, 0xc820479000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc82012fba0)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 2374 [select]:
google.golang.org/grpc/transport.(*http2Client).controller(0xc820508780)
	/go/src/google.golang.org/grpc/transport/http2_client.go:833 +0x5da
created by google.golang.org/grpc/transport.newHTTP2Client
	/go/src/google.golang.org/grpc/transport/http2_client.go:194 +0x153b

goroutine 1691 [select]:
net/http.(*persistConn).writeLoop(0xc82012edd0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 2348 [select]:
github.com/docker/engine-api/client/transport/cancellable.Do.func3(0x7f473391fab0, 0xc820514780, 0xc82035b050, 0xc8204007e0)
	/go/src/github.com/docker/engine-api/client/transport/cancellable/cancellable.go:78 +0xef
created by github.com/docker/engine-api/client/transport/cancellable.Do
	/go/src/github.com/docker/engine-api/client/transport/cancellable/cancellable.go:84 +0x336

goroutine 2204 [IO wait]:
net.runtime_pollWait(0x7f4733966e20, 0x72, 0xc82054f000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203b21b0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203b21b0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203b2150, 0xc82054f000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c320, 0xc82054f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc8201ce820)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1928 [select]:
net/http.(*persistConn).writeLoop(0xc82012fba0)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 2366 [select]:
google.golang.org/grpc.(*Conn).transportMonitor(0xc8200c35e0)
	/go/src/google.golang.org/grpc/clientconn.go:511 +0x1d3
google.golang.org/grpc.NewConn.func1(0xc8200c35e0)
	/go/src/google.golang.org/grpc/clientconn.go:322 +0x1b5
created by google.golang.org/grpc.NewConn
	/go/src/google.golang.org/grpc/clientconn.go:323 +0x509

goroutine 1563 [IO wait]:
net.runtime_pollWait(0x7f4733922c20, 0x72, 0xc82022a000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203d32c0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203d32c0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203d3260, 0xc82022a000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c368, 0xc82022a000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc820132340)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 1801 [IO wait]:
net.runtime_pollWait(0x7f4733922e60, 0x72, 0xc8203a9000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203d23e0, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203d23e0, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203d2380, 0xc8203a9000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c350, 0xc8203a9000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc82012f380)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 2372 [chan receive]:
github.com/cockroachdb/cockroach/rpc.NewContext.func1()
	/go/src/github.com/cockroachdb/cockroach/rpc/context.go:88 +0x57
github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc8202aecb0, 0xc82023ede0)
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:139 +0x52
created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:140 +0x62

goroutine 2102 [IO wait]:
net.runtime_pollWait(0x7f4733967120, 0x72, 0xc820467000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8202b0220, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8202b0220, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8202b01c0, 0xc820467000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c280, 0xc820467000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
net/http.(*persistConn).readLoop(0xc820246b60)
	/usr/local/go/src/net/http/transport.go:1069 +0x177
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 2346 [select]:
net/http.(*persistConn).readLoop(0xc8202472b0)
	/usr/local/go/src/net/http/transport.go:1178 +0xd52
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:853 +0x10a6

goroutine 2334 [IO wait]:
net.runtime_pollWait(0x7f47339229e0, 0x72, 0xc82049e000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8202ae220, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8202ae220, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8202ae1c0, 0xc82049e000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82002a1e0, 0xc82049e000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4
--
github.com/cockroachdb/cockroach/acceptance/cluster.(*LocalCluster).monitor(0xc8201ce0d0)
	/go/src/github.com/cockroachdb/cockroach/acceptance/cluster/localcluster.go:568 +0xd1
created by github.com/cockroachdb/cockroach/acceptance/cluster.(*LocalCluster).Start
	/go/src/github.com/cockroachdb/cockroach/acceptance/cluster/localcluster.go:592 +0x731

goroutine 2326 [select]:
net/http.(*persistConn).writeLoop(0xc82031e000)
	/usr/local/go/src/net/http/transport.go:1273 +0x472
created by net/http.(*Transport).dialConn
	/usr/local/go/src/net/http/transport.go:854 +0x10cb

goroutine 2325 [IO wait]:
net.runtime_pollWait(0x7f4733966b20, 0x72, 0xc8203a6000)
	/usr/local/go/src/runtime/netpoll.go:160 +0x60
net.(*pollDesc).Wait(0xc8203d2220, 0x72, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc8203d2220, 0x0, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc8203d21c0, 0xc8203a6000, 0x1000, 0x1000, 0x0, 0x7f4733959050, 0xc820014200)
	/usr/local/go/src/net/fd_unix.go:250 +0x23a
net.(*conn).Read(0xc82010c3f8, 0xc8203a6000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/go/src/net/net.go:172 +0xe4

```
Please assign, take a look and update the issue accordingly.

--- a/build/circle-test.sh
+++ b/build/circle-test.sh
@@ -150,7 +150,7 @@ if is_shard 0; then
     # this one runs outside the builder container (and inside the
     # container, something is already combining stdout and stderr).
     time $(dirname $0)/../acceptance.test -nodes 3 -l ${outdir}/acceptance \
-      -test.v -test.timeout 5m \
+      -test.v -test.timeout 10m \
       --verbosity=1 --vmodule=monitor=2 2>&1 | \
       tr -d '\r' | tee "${outdir}/acceptance.log" | \
       grep -E "^\--- (PASS|FAIL)|^(FAIL|ok)|${match}" |

--- a/circle.yml
+++ b/circle.yml
@@ -44,7 +44,7 @@ deployment:
       - aws configure set region us-east-1
       - build/build-static-binaries.sh
       - mkdir -p "${CIRCLE_ARTIFACTS}/acceptance_deploy"
-      - time acceptance/acceptance.test -test.v -test.timeout 5m
+      - time acceptance/acceptance.test -test.v -test.timeout 10m
           -i cockroachdb/cockroach -nodes 3
           -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
           "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"
@@ -63,7 +63,7 @@ deployment:
       - aws configure set region us-east-1
       - build/build-static-binaries.sh
       - mkdir -p "${CIRCLE_ARTIFACTS}/acceptance_deploy"
-      - time acceptance/acceptance.test -test.v -test.timeout 5m
+      - time acceptance/acceptance.test -test.v -test.timeout 10m
           -i cockroachdb/cockroach -nodes 3
           -l "${CIRCLE_ARTIFACTS}"/acceptance_deploy 2>&1 >
           "${CIRCLE_ARTIFACTS}/acceptance_deploy.log"

